### DISCUSSION TITLE
What do the different events do exactly?

### DISCUSSION THREAD

**Discussion by Fallensamurai94** [TEHE-Studios/ExpandedHelicopterEvents#149] [Q&A] [answered]
From the available events you can change listed below I have not been able to find exactly what any of these events are or what they do could someone explain them?

        Frequency_jet = 3,
        Frequency_police = 3,
        Frequency_news_chopper = 3,
        Frequency_military = 3,
        Frequency_FEMA_drop = 3,
        Frequency_samaritan_drop = 3,
        Frequency_survivor_heli = 3,
        Frequency_raiders = 3,
        Frequency_IRS = 3,
        Frequency_Spiffocopter = 3,
        Frequency_TISCreamery = 3,
        Frequency_UFO = 3,

**Comment by DimiDerevo** [comment, marked as answer]
There is description in Edit Settings 

JETS - Jets fly by players causing horde movement
POLICE - Police will take matters into their own hands and escort citizens around while firing zombies
NEWS - News choppers will hone in on any citizen they find and follow them around for a short time (looks like vanilla heli)
MILITARY - The military will progress through stages: warn citizens, purge the undead, then ultimately purge anything that moves
FEMA AID DROPS - FEMA will conduct aid drops early on in the apocalypse
SAMARITANS - Samaritans will drop off supplies to help survivors
SURVIVORS - Survivors will only flyby on scouting missions causing horde movement
RAIDERS - Raiders will hone in on survivors to torment them for some entertainment

![image](https://user-images.githubusercontent.com/74081568/228860092-eb6a4695-b438-4d17-8443-1895542b0a60.png)
